package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bynder/bynder-js-sdk",
-  "version": "2.3.1",
+  "version": "2.3.3",
```

Comment:
any reason we skipped `2.3.2` ?

Reply:
Because there's already a tag and a release on GitHub with it. This would be a new one.